helm_deploy/prisoner-content-hub-frontend/templates/ingress.yaml

```diff
@@ -43,7 +43,7 @@ spec:
     {{- end }}
   {{- end }}
   {{- with .Values.ingress.host }}
-    - host: {{ tpl .pattern (dict "qualifier" $.Values.ingress.qualifier "Template" $.Template) }}
+    - host: {{ tpl .pattern (dict "Template" $.Template) }}
```

Comment:
I think places where you do tpl but only pass it `(dict "Template" $.Template)` could be replaced with a direct reference to something in `.Values`

Reply:
These were the bits that blew up when I tried to directly replace as it was complaining about invalid values, maps etc.